### DISCUSSION TITLE
warn Multiple Podfiles were found

### DISCUSSION THREAD

**Discussion by mithatb** [callstack/react-native-visionos#98] [Q&A] [answered]
Hello,

When i run "npm run visionos" i am getting below warning. Any idea how to solve this warning?

`warn Multiple Podfiles were found: ios/Podfile,visionos/Podfile. Choosing ios/Podfile automatically. If you would like to select a different one, you can configure it via "project.ios.sourceDir". You can learn more about it here: https://github.com/react-native-community/cli/blob/main/docs/configuration.md`

Thank you,

**Comment by okwasniewski** [comment, marked as answer]
Hey, this warning is not accurate for out-of-tree platforms. I'll fix it in CLI in upcoming versions so you can safely ignore it for now.
